src/zeep/client.py

```diff
@@ -77,14 +77,16 @@ def __getitem__(self, key):
 class Client(object):
 
     def __init__(self, wsdl, wsse=None, transport=None,
-                 service_name=None, port_name=None, plugins=None):
+                 service_name=None, port_name=None, plugins=[]):
```

Comment:
Thanks. This is one thing I forget again and again over the ages. Just doesn't stick.